docs/TextRegion.md

```diff
@@ -0,0 +1,86 @@
+* align (Align/str, optional) - the horizontal alignment of the text in the region. (Default: Align.L)
+* line_height (float, optional) - This is a factor by which the line spacing will be different from the font height. It works similar to the attribute of the same name in HTML/CSS. (default: 1.0)
+* print_sh (bool, optional) - Treat a soft-hyphen (\\u00ad) as a printable character, instead of a line breaking opportunity. (Default: False)
+* skip_leading_spaces (default: False) - This flag is primarily used by `write_html()`, but may also have other uses. It removes all space characters at the beginning of each line.
+* wrapmode (default "WORD") - 
+
+All of those values can be overriden for each individual paragraph.
+
+
+### Common methods ###
+
+* paragraph() [see characteristics parameters below] - establish a new paragraph in the text. The text added to this paragraph will start on a new line.
+* write(text: str, link: = None) - write text to the region. This is only permitted when no explicit paragraph is currently active.
+* ln(h: float = None) - Start a new line moving either by the current font height or by the parameter "h". Only permitted when no explicit paragraph is currently active.
+* render() - if the region is not used as a context manager with "with", this method must be called to actually process the added text.
+
+
+## Paragraphs ##
+
+The primary purpose of paragraphs is to enable variations in horizontal text alignment, while the horizontal extents of the text are managed by the text region. To set the alignment, you can use the `align` argument when creating the paragraph, with the same `Align` values as elsewhere in the library.
+
+Note that the `write()` methods of paragraphs and text regions in general don't accept any other argument than "text" and "link".
+
+For more typographical control, you can use the following arguments. Most of those override the settings of the current region when set, and default to the value set there.
+
+* align (Align, optional) - The horizontal alignment of the paragraph.
+* line_height (float, optional) - factor by which the line spacing will be different from the font height. (default: by region) 
+* top_margin (float, optional) -  how much spacing is added above the paragraph. No spacing will be added at the top if the paragraph if the current y position is at (or above) the top margin of the page. (Default: 0.0)
+* bottom_margin (float, optional) - Those two values determine how much spacing is added below the paragraph. No spacing will be added at the bottom if it would result in overstepping the bottom margin of the page. (Default: 0.0)
+* skip_leading_spaces (float, optional) - removes all space characters at the beginning of each line.
+* wrapmode (WrapMode, optional)
```

Comment:
Again, this really looks like the docstring for `Paragraph.__init__` 😅

Would you be OK to replace this section by a link to <https://py-pdf.github.io/fpdf2/fpdf/text_region.html#fpdf.text_region.Paragraph>?

Reply:
There may be some differences between the signatures of `Paragraph.__init__()` and the `<Region>.paragraph()` that the user actually calls (and even that may theoretically vary depending on the type of region). Another one to revisit later, I guess.

Reply:
I fear this may not be revisited after this PR is merged,
but I'll follow your judgement about this 😊